src/test/java/com/snowflake/kafka/connector/records/MetaColumnTest.java

```diff
@@ -114,6 +120,17 @@ public void testConfig() throws IOException {
     assert result.get(META).has(RecordService.PARTITION);
     assert result.get(META).has(record.timestampType().name);
 
+    // test metadata configuration -- remove version
```

Comment:
this is for default, can you add a test for what happens when you add true to the metadata version? 

Also, once you are done, if you dont mind, I would like to add an e2e test here to verify if this works fine. 

Then we can push the code. (I will open another PR where you will be seen as co-author)